content/articles/2018-02-usethis-1-3-0.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: usethis 1.3.0
+slug: usethis-1-3-0
+description: >
+  usethis 1.3.0 is now on CRAN.
+author: Jenny Bryan
+date: 2018-02-23
+categories:
+  - package
+tags: []
+photo:
+  url: https://pixabay.com/en/tools-vintage-woodworking-saw-1209764/
+  author: unknown
+---
+
+```{r setup, include = FALSE}
+library(usethis)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are pleased to announce that [usethis](http://usethis.r-lib.org) 1.3.0 is now available on CRAN. usethis automates many common setup tasks for R projects, especially, but not only, for R packages. You can learn more about it at <http://usethis.r-lib.org>. Install it with:
+
+```{r, eval = FALSE}
+install.packages("usethis")
+```
+
+usethis is part of the refactoring of the [devtools](https://github.com/r-lib/devtools#readme) package and was [first released in late 2017](https://www.tidyverse.org/articles/2017/11/usethis-1.0.0/), at version 1.0.0. We say a bit more about its past and future below and then highlight notable developments in this version. For the full details, see the [release notes](https://github.com/r-lib/usethis/releases/tag/v1.3.0).
+
+## usethis vs. devtools
+
+[devtools](https://github.com/r-lib/devtools#readme) is an R package that helps you create, develop, and install R packages. By 2017, it was clear that devtools would be easier to maintain if it was split into a few smaller and more focused packages. usethis is the first of these to be released publicly on CRAN. usethis is the home for functions related to package and project setup.
```

Comment:
Also because devtools was trying to serve two radically opposing aims: a battery-included install so you could easily develop packages locally, and a minimal-dependency installer and checker (e.g. for travis)